vng_api_common/caching/etags.py

```diff
@@ -157,7 +157,8 @@ def mark_affected(cls, obj: models.Model, using=None) -> None:
         connection = transaction.get_connection(using)
 
         func = MethodCallback(etag_update.calculate_new_value)
-        for _, _func in connection.run_on_commit:
+        for run_on_commit in connection.run_on_commit:
+            _func = run_on_commit[1]
```

Comment:
Why is this changed needed?

Reply:
`run_on_commit` is private Django API, and change to be a two-tuple to a three-tuple. The previous `for x, y in ...` did not work anymore

Reply:
Thanks for the explanation!